packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -353,9 +349,28 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
     ],
     dropdownItemsWrapper: { selectors: { '&:focus': { outline: 0 } } },
     dropdownItems: [globalClassnames.dropdownItems, { display: 'block' }],
-    dropdownItem: [...dropdownItemStyle, itemSelectors()],
+    dropdownItem: [
+      ...dropdownItemStyle,
+      itemSelectors(),
+      {
+        selectors: {
+          [HighContrastSelector]: {
+            border: 'none',
+          },
+        },
+      },
+    ],
     dropdownItemSelected: dropdownItemSelected,
-    dropdownItemDisabled: dropdownItemDisabled,
+    dropdownItemDisabled: [
+      dropdownItemDisabled,
+      {
+        selectors: {
+          [HighContrastSelector]: {
+            border: 'none',
+          },
+        },
+      },
```

Comment:
same here, can go in `const dropdownItemDisabled` (defined above) and revert changes here